ansible/roles/test/tasks/sonic.yml

```diff
@@ -1,3 +1,5 @@
+- include_vars: "vars/docker_registry.yml"
```

Comment:
@wangxin do we need also to increase facts expiration timeout for this change?

Reply:
No, I don't think so. Vars are handled differently with facts. These are vars. If I remember correctly, they will not be affected by the facts expiration timer.

Reply:
This PR looks good. Is it actually used here? Can we remove it completely from roles/test?

---
In reply to: [380175312](https://github.com/Azure/sonic-mgmt/pull/1391#discussion_r380175312) [](ancestors = 380175312)

Reply:
Searched the code base:
* Variable defined in `ansible/vars/docker_registry.yml` is used by: `ansible\roles\sonic-common\tasks\sonicdocker.yml`.
* The `sonicdocker.yml` is used by `ansible/roles/test/tasks/saiserver.yml`

So, it would be better to keep it for now.